wazuh/wazuh-kibana-app
# [Feature] Mitre Att&ck Intelligence + adapt Framework

### Description
- Added new `Intelligence` tab to the `Mitre Att&ck` module with information about the Mitre resources: groups, mitigations, tactics, techniques using the new Wazuh API endpoints.
- Adapt Framework tab to the new Wazuh API endpoints.
 
### How to test
- Intelligence tab:
  - Each Mitre resource should be navigable when clicking on its label
  - Each Mitre resource should display a table with information and the table should:
     - Paginate
     - Sort
     - Open the details in a flyout with more information about the selected resource
     - Search bar:
        - Display suggestions and works the searching
        - Let searching with a term
  - Global search:
    - Let searching with a term and display the results by Mitre resource. It only displays the top 5 results, if there are more of them, a button with `See more results` label is displayed.
      - Click on the result button: open the resource details in a flyout
      - Click on the `See more results` button, navigate to the resource and add a filter (in the search bar) and display a table with the results
- Intelligence detail flyout
  - Display basic information about the resource
  - Display another information in tables.
  - Click on another resource should refresh the details flyout with the information of the selected resource
  
- Framework
  - The Tactics and Techniques panels should display.
  - Open a technique should open a flyout:
    - Click on a technique should navigate to the Intelligence section and the selected resource as a filter in the resource search bar
    - Click on a rule should navigate the Rules with the selected rule
    
Limitations:
- The resource tables can't be sorted by `ID` due to a Wazuh API limitation https://github.com/wazuh/wazuh/issues/8879
Closes #3202
